docs/docs/external/getting-started/interop/data-sharing.md

```diff
@@ -12,14 +12,51 @@ title: Data Sharing
 This document lists sample ways in which you can exercise the data-sharing interoperation protocol on the test network [launched earlier](../test-network/overview).
 
 Once the networks, relays, and drivers have been launched, and the ledgers bootstrapped, you can trigger three different interoperation flows corresponding to distinct data-sharing combinations as follows:
```

Comment:
"three different interoperation flows" --> "interoperation flows in four distinct configurations"